### ISSUE TITLE
USER ISSUE: can no longer play with friends

### ISSUE THREAD

**Issue by Goddessmeg** [StrangeLoopGames/EcoIssues#25052]
\
**User Id: (01e5c98a-5cef-45f3-c868-08dcbd8319be)**
**Version:** 0.11.0.0 beta release-693

**Steps to Reproduce:**
cant play on any world we make new or any old world we where playing before update 

**Expected behavior:**
we where able to plat together with no issues before update 

**Actual behavior:**
The following error message popped up:
```
The requested server was not found (ID = 4f7af241-f507-48f3-b158-ae11c6af1b9e). It either doesn't exist, or is not registered on the master server.
```

**Do you have mods installed? Does the issue happen when no mods are installed?:**
no mods 

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by JakeKoekemoer** [comment]
I am having the same issue but its also happening to me when I join any server on the server list or my friends try to join on me.

Also have no mods installed

**Comment by D3nnis3n** [comment]
This should be addressed with Hotfix 11.0.1, if the issue persists please create a new issue and attach your logfiles as noted in the issue template.
